code/modules/admin/verbs/striketeam.dm

```diff
@@ -1,31 +1,46 @@
-//STRIKE TEAMS
+//Deathsquad
```

Comment:
Might be a good idea to rename the whole file to `deathsquad.dm`
(do take care though, it's easy to slighly mess up both git (use `git mv`, not the windows standard "rename file" functionality) and dm (adjust the path in `paradise.dme`)